packages/block-library/src/file/block.json

```diff
@@ -70,7 +70,10 @@
 				"link": true
 			}
 		},
-		"interactivity": true
+		"interactivity": true,
+		"interactivity": {
+			"clientNavigation": true
+		}
```

Comment:
This one is already interactive.

```suggestion
		"interactivity": true
```

Reply:
And was duplicated, my bad. Similar for search and navigation.